config/locales/en.yml

```
@@ -450,6 +450,12 @@ en:
         edit:
           delete_link: I understand the consequences, proceed to delete '%{name}' backend
           delete_confirmation: Are you sure you want to delete the backend '%{name}'?
+      bot_protections:
+        form:
+          title: Protection against bots on %{portal} portal
```

```suggestion
        form:
          title: Protection against bots on %{portal} portal
```

I think this is not needed anymore

---

Right